pmm-tests/minikube_operators_setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Prepare a set of base64 encoded values and non encoded for user and pass with administrator privileges to pmm-server (DBaaS)
+PMM_USER='admin';
+PMM_PASS='admin';
+OPERATOR_VERSION=$1;
```

Comment:
For the sake of better DX I suggest we check if the variable is actually passed, and error out if not:
`if [ -z "$OPERATOR_VERSION" ] ; then
    echo "Error: no parameter passed"
    echo "Use: bash ./minikube_operators_setup.sh <operator_version>"
    exit 1
fi
`

Reply:
fixed